src/test/kotlin/com/github/hanseter/json/editor/base/EditorTestBase.kt

```diff
@@ -0,0 +1,51 @@
+package com.github.hanseter.json.editor.base
+
+import com.github.hanseter.json.editor.JsonPropertiesEditor
+import com.github.hanseter.json.editor.ui.ControlTreeItemData
+import com.github.hanseter.json.editor.ui.CustomTreeTableView
+import com.sun.javafx.robot.FXRobot
+import javafx.scene.Scene
+import javafx.scene.control.TreeTablePosition
+import javafx.scene.input.KeyCode
+import javafx.stage.Stage
+import org.json.JSONObject
+import org.junit.jupiter.api.extension.ExtendWith
+import org.testfx.api.FxRobot
+import org.testfx.framework.junit5.ApplicationExtension
+import org.testfx.framework.junit5.Start
+import org.testfx.util.WaitForAsyncUtils
+
+@ExtendWith(ApplicationExtension::class)
+open class EditorTestBase {
+
+    lateinit var editor: JsonPropertiesEditor
+    lateinit var robot: FxRobot
```

Comment:
If I'm not mistaken testfx preferred way to acquire a reference to a robot is by injecting an instance of it into the test by simply adding it as a parameter.